src/interface/src/app/treatments/map-project-areas/map-project-areas.component.ts

```diff
@@ -59,21 +55,21 @@ export class MapProjectAreasComponent implements OnInit {
   scenarioId = this.treatmentsState.getScenarioId();
   summary$ = this.treatmentsState.summary$;
   mouseLngLat: LngLat | null = null;
-  fillColor: any;
+  fillColor!: any;
   getTreatedStandsTotal = getTreatedStandsTotal;
 
-  hoveredProjectAreaId$ = new Subject<number>();
```

Comment:
Hey @lastminutediorama  that do you think about this Change? Since we needed the rank for hovering purposes now I'm storing all the MapGeoJSONFeature object and using the id or the rank from there.
Can you please take a look just in case?
Thank you!

Reply:
Maybe we still need the hoveredProjectAreaId$ since we are using that in the combineLatest?

Reply:
I think the only real reason we get the `hoveredProjectAreaId$` and then `hoveredProjectArea$` is for the tooltip, so we can have the prescription data to [display the total stand count here](https://github.com/OurPlanscape/Planscape/blob/d3406d7fb70edbdb79f650f608957b841e4f5e91/src/interface/src/app/treatments/map-project-areas/map-project-areas.component.html#L53).

If there's another way to get that prescription data from the hoveredProjectAreaFromFeatures, then we probably don't need to set the hoveredProjectAreaId$ from the hovered stand, or use that to set up the ProjectArea record.

Reply:
I think the only real reason we get the hoveredProjectAreaId$ and then hoveredProjectArea$ is for the tooltip, so we can have the prescription data to [display the total stand count here](https://github.com/OurPlanscape/Planscape/blob/d3406d7fb70edbdb79f650f608957b841e4f5e91/src/interface/src/app/treatments/map-project-areas/map-project-areas.component.html#L53).

If there's another way to get that prescription data from the hoveredProjectAreaFromFeatures, then we probably don't need to set the hoveredProjectAreaId$ from the hovered stand, or use that to set up the ProjectArea record.